mri/gridsearch.py

```diff
@@ -16,7 +16,6 @@
 import psutil
 
 # Package import
-from mri.numerics.utils import generate_operators
```

Comment:
Is this change needed for the documentation to be fixed?
If so why?

Reply:
Yes, we no longer have this import, and in trying to figure out all modules, documentation errors out.

Reply:
I see